bin/diagnostic.js

```diff
@@ -1 +1,23 @@
 'use strict';
+```js
+
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ");
+  if (answer <1 || >10) {
+   console.log("Needs to be between 1 and 10!");
+   break;
+ }else {
+   console.log("You guessed it!");
+  }
+}
+```
```

Comment:
Here's one possible solution. Take a look at this and see if anything is unclear.
```
const ask = require('../lib/ask.js');

let answer = '';

while (answer !== '5') {
  if (answer < 1 || answer > 10) {
   console.log('Please try a number between 1 and 10');
 } else if (answer !== '') {
   console.log('That\'s not it!');
 }
  answer = ask("Guess my secret? ");
}
console.log("You guessed it!");
```